util/tests/timer.cc

```diff
@@ -0,0 +1,41 @@
+#include <catch.hpp>
+#include <leatherman/util/timer.hpp>
+
+namespace leatherman { namespace util {
+
+SCENARIO("Using Timer", "[util]") {
+    SECTION("can instantiate") {
+        REQUIRE_NOTHROW(Timer());
+    }
+
+    SECTION("can reset") {
+        Timer t {};
+        REQUIRE_NOTHROW(t.reset());
+    }
+
+    SECTION("can retrieve durations [s]") {
+        Timer t {};
+        auto d1_s = t.elapsed_seconds();
+        auto d2_s = t.elapsed_seconds();
+
+        REQUIRE(d1_s <= d2_s);
+    }
+
+    SECTION("can retrieve durations [s]") {
+        Timer t {};
+        auto d1_ms = t.elapsed_milliseconds();
```

Comment:
Ok, I'll add short pauses, otherwise:
```
/Users/alessandro/code/leatherman/util/tests/timer.cc:29: FAILED:
  REQUIRE( d1_ms < d2_ms )
with expansion:
  0 < 0
```